src/tabular-api-surface.ts

```diff
@@ -212,376 +203,10 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
-
-      // Retry on "received rst stream" errors
-      const isRstStreamError = (error: ServiceError): boolean => {
-        if (error.code === 13 && error.message) {
-          const error_message = (error.message || '').toLowerCase();
-          return (
-            error.code === 13 &&
-            (error_message.includes('rst_stream') ||
-              error_message.includes('rst stream'))
-          );
-        }
-        return false;
-      };
-
-      metricsCollector.handleStatusAndMetadata(requestStream);
-      rowStream
-        .on('error', (error: ServiceError) => {
-          rowStreamUnpipe(rowStream, userStream);
-          activeRequestStream = null;
-          if (IGNORED_STATUS_CODES.has(error.code)) {
-            // We ignore the `cancelled` "error", since we are the ones who cause
-            // it when the user calls `.abort()`.
-            userStream.end();
-            metricsCollector.onOperationComplete(error.code);
-            return;
-          }
-          numConsecutiveErrors++;
-          numRequestsMade++;
-          if (
-            numConsecutiveErrors <= maxRetries &&
-            (RETRYABLE_STATUS_CODES.has(error.code) ||
-              isRstStreamError(error)) &&
-            !(timeout && timeout < new Date().getTime() - callTimeMillis)
-          ) {
-            const backOffSettings =
-              options.gaxOptions?.retry?.backoffSettings ||
-              DEFAULT_BACKOFF_SETTINGS;
-            const nextRetryDelay = getNextDelay(
-              numConsecutiveErrors,
-              backOffSettings,
-            );
-            metricsCollector.onAttemptComplete(error.code);
-            retryTimer = setTimeout(makeNewRequest, nextRetryDelay);
-          } else {
-            if (
-              !error.code &&
-              error.message === 'The client has already been closed.'
-            ) {
-              //
-              // The TestReadRows_Generic_CloseClient conformance test requires
-              // a grpc code to be present when the client is closed. The
-              // appropriate code for a closed client is CANCELLED since the
-              // user actually cancelled the call by closing the client.
-              //
-              error.code = grpc.status.CANCELLED;
-            }
-            metricsCollector.onOperationComplete(error.code);
-            userStream.emit('error', error);
-          }
-        })
-        .on('data', _ => {
-          // Reset error count after a successful read so the backoff
-          // time won't keep increasing when as stream had multiple errors
-          numConsecutiveErrors = 0;
-          metricsCollector.onResponse();
-        })
-        .on('end', () => {
-          activeRequestStream = null;
-          metricsCollector.onOperationComplete(grpc.status.OK);
-        });
-      rowStreamPipe(rowStream, userStream);
-    };
-
-    makeNewRequest();
-    return userStream;
+    return createReadStreamInternal(this, true, opts);
```

Comment:
why is this true? I think createReadStream is for scan not reading a single row?